src/ray/raylet/scheduling/cluster_task_manager.cc

```diff
@@ -1399,5 +1463,29 @@ ResourceSet ClusterTaskManager::CalcNormalTaskResources() const {
   return ResourceSet(total_normal_task_resources);
 }
 
+uint64_t ClusterTaskManager::MaxRunningTasksPerSchedulingClass(
+    SchedulingClass sched_cls_id) const {
+  auto sched_cls = TaskSpecification::GetSchedulingClassDescriptor(sched_cls_id);
+  double cpu_req = sched_cls.resource_set.GetNumCpusAsDouble();
+  uint64_t total_cpus = cluster_resource_scheduler_->GetNumCpus();
+
+  if (cpu_req == 0 || total_cpus == 0) {
+    return std::numeric_limits<uint64_t>::max();
+  }
+  return static_cast<uint64_t>(std::round(total_cpus / cpu_req));
+}
+
+void ClusterTaskManager::UpdateSchedulingClassWaitTime(
+    SchedulingClassInfo &sched_cls_info) {
+  if (sched_cls_info.running_tasks.size() >= sched_cls_info.capacity) {
```

Comment:
Can you remove this method? Generally if there is a method that only gets called once, I think it's nicer to be able to read the code where it's used.